### PR TITLE
feat: iiif/iip/ visualization, batch import, angular 17, ui redesign

### PR DESCRIPTION
**Pull request by MyleneSimon**
## What does this PR do?

- batch import of collections
- thumbnails and iiif/iip visualizations
- home page
- upgrade to angular 17
- migrate from NgBootstrap and Angular Material to PrimeNG
- minor fixes
- remove deprecated views
- upgrade wdzt to custom version

### CODE REVIEW

File: src/app/app.component.css [hunk 1]
```diff
@@ -46,17 +46,35 @@ a:not([href]).dropdown-toggle:hover {
...
-  margin: 0;
-  padding-bottom: 6rem;
-  min-height: 100%;
-}
+/*body {*/
+/*  position: relative;*/
+/*  margin: 0;*/
+/*  padding-bottom: 6rem;*/
+/*  min-height: 100%;*/
+/*}*/
```
Comment (antoinemeyer5) [inline src/app/app.component.css]: L72 to L77: can be removed?

File: src/app/app.component.css [hunk 2]
```diff
@@ -66,7 +84,15 @@ footer {
...
   border-top: 1px solid #e7e7e7;
+  /*position: fixed;*/
 }
 
+/*.layout-footer {*/
+/*  position: fixed;*/
+/*  bottom: 0;*/
+/*  width: 100%;*/
+/*  z-index: 1;*/
+/*}*/
```
Comment (antoinemeyer5) [inline src/app/app.component.css]: L92 to L97: can be removed?

File: src/app/app.component.ts [hunk 3]
```diff
@@ -17,8 +19,87 @@ export class AppComponent {
...
+        label: 'Plugins',
+        icon: 'pi pi-box',
+        routerLink: '/plugins'
+      },
+      {
+        label: 'Workflows',
+        icon: 'pi pi-share-alt',
+        routerLink: '/workflows'
+      }
+    ];
```
Comment (antoinemeyer5) [inline src/app/app.component.ts:102]: L39 to L102: This can be put in a separate file, right?

File: src/app/home/home.component.html
Comment (antoinemeyer5) [inline src/app/home/home.component.html:1]: Commented parts of the code can be deleted?

File: src/app/workflow/widgets/search-widget/search-widget.component.ts [hunk 4]
```diff
@@ -17,32 +18,24 @@ export class SearchWidgetComponent extends StringWidget {
   public data: Array<any>;
   public service: any;
 
+  //TODO fix widget
```
Comment (antoinemeyer5) [inline src/app/workflow/widgets/search-widget/search-widget.component.ts]: Something to finish here?

File: src/app/workflow/workflow-detail/workflow-detail.component.html [hunk 5]
```diff
@@ -1,213 +1,300 @@
...
+<!--    <button *ngIf="!workflow.publiclyShared && workflow.status === 'SUCCEEDED' && canEdit()" type="button"-->
+<!--            ngbTooltip="Make workflow public"-->
+<!--            placement="left"-->
+<!--            class="btn btn-light" (click)="makeWorkflowPublic();">-->
+<!--          <span class="fa fa-users"-->
+<!--                aria-hidden="true"></span>-->
+<!--    </button>-->
+
+<!--  </div>-->
+<!--</div>-->
```
Comment (antoinemeyer5) [inline src/app/workflow/workflow-detail/workflow-detail.component.html]: L92 to L151: Something to finish? Or maybe commented code can be remove right?

File: src/app/pyramid-visualization/pyramid-visualization-new/pyramid-visualization-new.component.html [hunk 6]
```diff
@@ -1,28 +1,38 @@
...
+<!--    </div>-->
+<!--  </form>-->
+<!--</div>-->
+<!--<div class="modal-footer">-->
+<!--  <button type="button" class="btn btn-danger" (click)="cancel()">Cancel</button>-->
+<!--  <button type="button" class="btn btn-success" (click)="save()"-->
+<!--          [disabled]="">-->
+<!--    Save-->
+<!--  </button>-->
+<!--</div>-->
```
Comment (antoinemeyer5) [inline src/app/pyramid-visualization/pyramid-visualization-new/pyramid-visualization-new.component.html]: L14 to L37: Should this commented code remain?

File: src/app/stitching-vector/stitching-vector-detail/stitching-vector-detail.component.ts [hunk 7]
```diff
@@ -41,32 +41,44 @@ export class StitchingVectorDetailComponent implements OnInit {
...
+    //       return this.stitchingVectorService.getTimeSlices(this.stitchingVectorId, params);
+    //     }),
+    //     map(paginatedResult => {
+    //       this.resultsLength = paginatedResult.page.totalElements;
+    //       return paginatedResult.data;
+    //     }),
+    //     catchError(() => {
+    //       return observableOf([]);
+    //     })
+    //   ).subscribe(data => this.timeSlices = data);
```
Comment (antoinemeyer5) [inline src/app/stitching-vector/stitching-vector-detail/stitching-vector-detail.component.ts]: L64 to L81: Cannot be removed?

File: src/app/stitching-vector/stitching-vector-list/stitching-vector-list.component.html
Comment (antoinemeyer5) [inline src/app/stitching-vector/stitching-vector-list/stitching-vector-list.component.html:1]: Many parts of this code are commented on, even though they are already equivocal: perhaps the commented-on parts can be deleted?

**Review by antoinemeyer5 — COMMENTED** [review]
LGTM, I just have a few comments on details

**Comment by antoinemeyer5** [comment]
The pop-up confirming deletion of a image collection has not been affected by migration.

Need to migrate this pop-up before merge! @MyleneSimon

**Comment by antoinemeyer5** [comment]
At the bottom right of every page is the word “logo” and an "i" information icon.
When clicked, a pop-up should appear.

Need to migrate this pop-up before merge!

**Comment by antoinemeyer5** [comment]
Pop-up confirmation of image collection deletion is not available on `WIPP-dev`, but is available in this PR! So nothing todo.

File: src/app/app.component.css (hunk 1 above)
Comment (MyleneSimon) [reply src/app/app.component.css]: Fixed

File: src/app/app.component.css (hunk 2 above)
Comment (MyleneSimon) [reply src/app/app.component.css]: Fixed

File: src/app/workflow/widgets/search-widget/search-widget.component.ts (hunk 4 above)
Comment (MyleneSimon) [reply src/app/workflow/widgets/search-widget/search-widget.component.ts]: Fixed

File: src/app/workflow/workflow-detail/workflow-detail.component.html (hunk 5 above)
Comment (MyleneSimon) [reply src/app/workflow/workflow-detail/workflow-detail.component.html]: Fixed

File: src/app/pyramid-visualization/pyramid-visualization-new/pyramid-visualization-new.component.html (hunk 6 above)
Comment (MyleneSimon) [reply src/app/pyramid-visualization/pyramid-visualization-new/pyramid-visualization-new.component.html]: Fixed

File: src/app/stitching-vector/stitching-vector-detail/stitching-vector-detail.component.ts (hunk 7 above)
Comment (MyleneSimon) [reply src/app/stitching-vector/stitching-vector-detail/stitching-vector-detail.component.ts]: Fixed

File: src/app/home/home.component.html
Comment (MyleneSimon) [reply src/app/home/home.component.html:1]: Fixed

File: src/app/stitching-vector/stitching-vector-list/stitching-vector-list.component.html
Comment (MyleneSimon) [reply src/app/stitching-vector/stitching-vector-list/stitching-vector-list.component.html:1]: Fixed

File: src/app/app.component.ts (hunk 3 above)
Comment (MyleneSimon) [reply src/app/app.component.ts:102]: Will be addressed in next iteration!

**Pull request merged**